lib/MusicBrainz/Server/Entity/Area.pm

```diff
@@ -103,10 +103,11 @@ around TO_JSON => sub {
     my ($orig, $self) = @_;
 
     my $json = {
-        %{ $self->$orig },
-        iso_3166_1_codes => [$self->iso_3166_1_codes],
-        iso_3166_2_codes => [$self->iso_3166_2_codes],
-        iso_3166_3_codes => [$self->iso_3166_3_codes],
+        %{$self->$orig},
+        iso_3166_1_codes => [ $self->iso_3166_1_codes ],
+        iso_3166_2_codes => [ $self->iso_3166_2_codes ],
+        iso_3166_3_codes => [ $self->iso_3166_3_codes ],
```

Comment:
You changed the code style here to add spaces inside the `[]` but remove spaces from inside the `%{}`, which doesn't seem consistent to me. :P (But if you want to change style, ideally do it in separate commits so that the functional changes are easier to see.)

Reply:
Sigh. IDE again. Will undo. At this rate I might have to work on vim.